policyengine_us/parameters/gov/states/mt/tax/income/subtractions/tuition/cap.yaml

```diff
@@ -0,0 +1,17 @@
+description: Montana caps the tuition subtraction at this maximum amount.
+values:
+  2021-01-01: 3_000
+metadata:
+  unit: currency-USD
+  period: year
+  label: Montana tuition subtraction cap
+  reference:
+  - title: 2023 MT Legal Code, Title 15, Section 30, Part 21, (5a)
+    href: https://leg.mt.gov/bills/mca/title_0150/chapter_0300/part_0210/section_0200/0150-0300-0210-0200.html
+  - title: 2021 MT Legal Code, Title 15, Section 30, Part 21, (5a)
+    href: https://leg.mt.gov/bills/2022/mca/title_0150/chapter_0300/part_0210/section_0200/0150-0300-0210-0200.html
+  - title: 2022 Montana Individual Income Tax Return
+    href: https://mtrevenue.gov/wp-content/uploads/dlm_uploads/2023/05/Montana-Idividiual-Income-Tax-Return-Form-2-2022v6.2.pdf
```

Comment:
delete this reference and find a tax form instructions that mentions the $3,000

Reply:
same for 2021 reference

Reply:
Having trouble finding this